### ISSUE TITLE
Feature Request: Folderlistmenu enhancement

### ISSUE THREAD

**Issue by CollinChaffin** [tablacus/TablacusExplorerAddons#41]
Hi Gaku,

I was finally able to properly configure this addon after you got the doc added (thanks!).  It looks like it can handle PART of what I had hoped.  I am ultimately trying to simply duplicate (as closely as possible) the functionality of the tabbed explorer addon/replacement (old) named Clover.  As did many others, I only ever found the early versions of that app usable (3.0.46 was last good version).  Anyway I finally just recorded a quick ~1 min video capture of the bookmark dropdown functionality I am looking for in Tablacus.  Can you help me to get there?  It looks like this folderlistmenu most closely provides SOME of the functionality but not all whereas the favorites and linkbar also provides only some.  I am looking to place not only shortcuts, but folders and app links in the fashion shown and drag and drop optional but as you can see makes things very nice.

On a side note, I am still trying to get Tablacus' tabs to look like these using the tabdesigner addon!

Thanks!

![snag-11-10-20185 58 40 am avi_compressed](https://user-images.githubusercontent.com/9301483/48301446-47063400-e4e6-11e8-91ec-121550522938.gif)

**Comment by tablacus** [comment]
I think this Japanese page will be helpful.
https://qiita.com/nocd5/items/af062f7b06d4cd3b8b18

Regards,

**Comment by CollinChaffin** [comment]
Hi Gaku,

I don't understand how is that helpful to my question?  Sorry I'm trying to translate but do not understand.

**Comment by duarteframos** [comment]
@CollinChaffin Would you mind sharing your user CSS file? I am just a potential user trying to figure out what Tablacus can do to see if it fits my needs, but I am sure you already noticed the lack of documentation.

In the process I've been gradually trying to improve the current Wiki from what I can learn here, and by looking at your CSS file I might just figure out a bunch of classes I could add to the [Addon Page](https://github.com/tablacus/TablacusExplorerAddons/wiki/usercss)

I know it is a personal thing, and it probably took a lot of work to build, so I understand if you don't want to share.

**Comment by ccfs** [comment]
duarteframos,

In case you have not noticed, I keep updating an add-on documentation which is published at https://github.com/ccfs/TablacusExplorer/issues/1. Hope you find it useful.

Regards,

**Comment by duarteframos** [comment]
Ah yes this is useful, thanks for the link.
I've been trying to improve the wiki based on what info I gather from around the issues here, I'll take a look at what info you gathered there.

Would it be ok if I included parts of it here in the wiki pages?

**Comment by ccfs** [comment]
Sure, actually most wiki page contents were taken from the pdf file, but the wiki pages have not been updated for a long time.

**Comment by duarteframos** [comment]
Yes I've been trying to improve on that. I'll try to make good use of what you gathered.
Thanks for the help :)
